app/views/layouts/application.html.erb

```diff
@@ -12,13 +12,12 @@
   </head>
 
   <body>
-    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
```

Comment:
[question] Is there a reason why you're moving the skipnav but not adding it back in somewhere else?

Reply:
i think it's in there twice. (i kept meaning to go back and fix it but haven't so thank you ryan!)

Reply:
Oh yep, that's totally it.  Cool :)